recipes/cairo/all/conanfile.py

```diff
@@ -0,0 +1,258 @@
+                shutil.copy(pc_name, os.path.join('pkgconfig', os.path.basename(pc_name)))
+            if self.options.with_freetype:
+                tools.replace_in_file(os.path.join(self.source_folder, self._source_subfolder, "src", "cairo-ft-font.c"),
+                                      '#if HAVE_UNISTD_H', '#ifdef HAVE_UNISTD_H')
+            env_build = self._get_env_build()
+            env_build.install()
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        if self._is_msvc:
+            src = os.path.join(self._source_subfolder, 'src')
+            cairo_gobject = os.path.join(self._source_subfolder, 'util', 'cairo-gobject')
+            inc = os.path.join('include', 'cairo')
+            self.copy(pattern="cairo-version.h", dst=inc, src=self._source_subfolder)
+            self.copy(pattern="cairo-features.h", dst=inc, src=src)
+            self.copy(pattern="cairo.h", dst=inc, src=src)
+            self.copy(pattern="cairo-deprecated.h", dst=inc, src=src)
+            self.copy(pattern="cairo-win32.h", dst=inc, src=src)
+            self.copy(pattern="cairo-script.h", dst=inc, src=src)
+            self.copy(pattern="cairo-ft.h", dst=inc, src=src)
+            self.copy(pattern="cairo-ps.h", dst=inc, src=src)
+            self.copy(pattern="cairo-pdf.h", dst=inc, src=src)
+            self.copy(pattern="cairo-svg.h", dst=inc, src=src)
+            self.copy(pattern="cairo-gobject.h", dst=inc, src=cairo_gobject)
+            if self.options.shared:
+                self.copy(pattern="*cairo.lib", dst="lib", src=src, keep_path=False)
+                self.copy(pattern="*cairo.dll", dst="bin", src=src, keep_path=False)
+                self.copy(pattern="*cairo-gobject.lib", dst="lib", src=cairo_gobject, keep_path=False)
+                self.copy(pattern="*cairo-gobject.dll", dst="bin", src=cairo_gobject, keep_path=False)
+            else:
+                self.copy(pattern="*cairo-static.lib", dst="lib", src=src, keep_path=False)
+                self.copy(pattern="*cairo-gobject.lib", dst="lib", src=cairo_gobject, keep_path=False)
+                shutil.move(os.path.join(self.package_folder, 'lib', "cairo-static.lib"),
+                            os.path.join(self.package_folder, 'lib', "cairo.lib"))
+        else:
+            with tools.chdir(self._source_subfolder):
+                env_build = self._get_env_build()
+                env_build.install()
+        tools.remove_files_by_mask(self.package_folder, "*.la")
+        
+        self.copy("COPYING*", src=self._source_subfolder, dst="licenses", keep_path=False)
+        tools.rmdir(os.path.join(self.package_folder, "lib", "pkgconfig"))
+
+
+    def package_info(self):
+        self.cpp_info.components["cairo_"].names["pkg_config"] = "cairo"
+        self.cpp_info.components["cairo_"].libs = ["cairo"]
+        self.cpp_info.components["cairo_"].includedirs = [os.path.join('include', 'cairo')]
```

Comment:
```suggestion
        self.cpp_info.components["cairo_"].includedirs.append(os.path.join('include', 'cairo'))
```

Reply:
the default includdirs has `include` in it, which we don't want. what is the gain here ?

Reply:
This is about having same behavior with a linux install.
On a default linux installation, cairo headers are installed in the default prefix `/usr`.
Somebody can thus do `#include <cairo/some_header.h>` without requiring to pass `-I/usr/include/cairo` to the compiler.